databroker/databroker_bin.cdx.json

```diff
@@ -0,0 +1,9900 @@
+{
```

Comment:
Intentional to commit Cyclone dx files? If so - how/when shall they be updated? Should possibly be documented. As they mention Databroker version I assume they need to be updated after you have updated version according to https://github.com/eclipse-kuksa/kuksa-databroker/wiki/Release-Process#update-rust-versions and possibly

FYI @SebastianSchildt

Reply:
Thank you, removed the unintentionally committed CycloneDX files.